src/framework/components/rigid-body/system.js

```diff
@@ -16,7 +16,13 @@ import { RigidBodyComponentData } from './data.js';
 let ammoRayStart, ammoRayEnd;
 
 /**
- * Object holding the result of a successful raycast hit.
+ * Contains the result of a successful raycast intersection with a rigid body. When a ray
+ * intersects with a rigid body in the physics simulation, this class stores the complete
+ * information about that intersection including the entity, the exact point of impact, the normal
```

Comment:
suggestion: impact -> intersection

Reply:
It is also fine to call the distance of the ray where the intersection occurred as time of impact, so both are ok here.